src/sets.jl

```diff
@@ -411,6 +458,11 @@ struct PositiveSemidefiniteConeSquare <: AbstractSymmetricMatrixSetSquare
     side_dimension::Int
 end
 
+function dual_set(s::PositiveSemidefiniteConeSquare)
+    return error("""Dual of $s is not defined because $s is not a proper cone. 
```

Comment:
I don't think this is the correct explanation. The discussion at square.jl says that "(P₊)* = D₊" where " D₊ [is] the cone of matrices A such that A+Aᵀ ∈ P₊" There's a well-defined dual of this cone, but it's not defined as an MOI set.